packages/react-dom-bindings/src/server/ReactDOMServerFormatConfig.js

```diff
@@ -178,7 +182,15 @@ export function createResponseState(
   bootstrapScriptContent: string | void,
   bootstrapScripts: $ReadOnlyArray<string | BootstrapScriptDescriptor> | void,
   bootstrapModules: $ReadOnlyArray<string | BootstrapScriptDescriptor> | void,
+  fallbackBootstrapScriptContent: string | void,
```

Comment:
( Nit / stylistic, feel free to disregard )

It's starting to feel that we're branching into many modes for `createResponseState`, with arguments that either (1) affect whether Fizz SSR has opted into a mode or (2) affect how Fizz behaves when in specific mode (e.g. `fallbackBootstrapScriptContent`)

I know that we prefer flat objects for perf reasons, and I can see why that would be useful for opaque objects that are read frequently e.g. `ResponseState`. However, would it be reasonable to change the interface for `createResponseState` and `renderIntoX` to use nested objects? These are only created / accessed once.

e.g. 
```
function createResponseState(
identifierPrefix: string | void,
  nonce: ...,
  bootstrapScriptContent: ...,
  bootstrapScripts: ...,
  bootstrapModules: ...,
  externalRuntimeConfig: ExternalRuntimeConfig | null,
  containerConfig: RenderIntoContainerConfig | null,
}